From 2b2e7aa7dfb3b41a2f0068a058bf7567465b87d2 Mon Sep 17 00:00:00 2001
From: David Tardon <dtardon@redhat.com>
Date: Mon, 25 Jun 2012 15:14:43 +0200
Subject: [PATCH] fix invalid .desktop files

desktop-file-validate issued the following error:

error: file contains group "New Shortcut Group", but groups extending
the format should start with "X-"

Change-Id: I45c0039ab788546a37780927284c8827a19fd52c
---
 sysui/desktop/menus/base.desktop    |    4 ++--
 sysui/desktop/menus/calc.desktop    |    4 ++--
 sysui/desktop/menus/draw.desktop    |    4 ++--
 sysui/desktop/menus/impress.desktop |    4 ++--
 sysui/desktop/menus/math.desktop    |    4 ++--
 sysui/desktop/menus/writer.desktop  |    4 ++--
 6 files changed, 12 insertions(+), 12 deletions(-)

diff --git a/sysui/desktop/menus/base.desktop b/sysui/desktop/menus/base.desktop
index 8cc8e7c..72cf467 100755
--- a/sysui/desktop/menus/base.desktop
+++ b/sysui/desktop/menus/base.desktop
@@ -10,8 +10,8 @@ Name=%PRODUCTNAME Base
 GenericName=Database Development
 Comment=Manage databases, create queries and reports to track and manage your information by using Base.
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=New
-[New Shortcut Group]
+X-Ayatana-Desktop-Shortcuts=X-New
+[X-New Shortcut Group]
 UnityQuicklist=New Database
 Exec=libreoffice --base %U
 TargetEnvironment=Unity
diff --git a/sysui/desktop/menus/calc.desktop b/sysui/desktop/menus/calc.desktop
index b1bde07..e9f3250 100755
--- a/sysui/desktop/menus/calc.desktop
+++ b/sysui/desktop/menus/calc.desktop
@@ -10,8 +10,8 @@ Name=%PRODUCTNAME Calc
 GenericName=Spreadsheet
 Comment=Perform calculations, analyze information and manage lists in spreadsheets by using Calc.
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=New
-[New Shortcut Group]
+X-Ayatana-Desktop-Shortcuts=X-New
+[X-New Shortcut Group]
 UnityQuicklist=New Spreadsheet
 Exec=libreoffice --calc %U
 TargetEnvironment=Unity
diff --git a/sysui/desktop/menus/draw.desktop b/sysui/desktop/menus/draw.desktop
index 0bec2b0..5c741a9 100755
--- a/sysui/desktop/menus/draw.desktop
+++ b/sysui/desktop/menus/draw.desktop
@@ -10,8 +10,8 @@ Name=%PRODUCTNAME Draw
 GenericName=Drawing Program
 Comment=Create and edit drawings, flow charts, and logos by using Draw.
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=New
-[New Shortcut Group]
+X-Ayatana-Desktop-Shortcuts=X-New
+[X-New Shortcut Group]
 UnityQuicklist=New Drawing
 Exec=libreoffice --draw %U
 TargetEnvironment=Unity
diff --git a/sysui/desktop/menus/impress.desktop b/sysui/desktop/menus/impress.desktop
index 0df250c..2fb8650 100755
--- a/sysui/desktop/menus/impress.desktop
+++ b/sysui/desktop/menus/impress.desktop
@@ -10,8 +10,8 @@ Name=%PRODUCTNAME Impress
 GenericName=Presentation
 Comment=Create and edit presentations for slideshows, meeting and Web pages by using Impress.
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=New
-[New Shortcut Group]
+X-Ayatana-Desktop-Shortcuts=X-New
+[X-New Shortcut Group]
 UnityQuicklist=New Presentation
 Exec=libreoffice --impress %U
 TargetEnvironment=Unity
diff --git a/sysui/desktop/menus/math.desktop b/sysui/desktop/menus/math.desktop
index 67f9ea6..122e01f 100755
--- a/sysui/desktop/menus/math.desktop
+++ b/sysui/desktop/menus/math.desktop
@@ -11,8 +11,8 @@ Name=%PRODUCTNAME Math
 GenericName=Formula Editor
 Comment=Create and edit scientific formulas and equations by using Math.
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=New
-[New Shortcut Group]
+X-Ayatana-Desktop-Shortcuts=X-New
+[X-New Shortcut Group]
 UnityQuicklist=New Formula
 Exec=libreoffice --math %U
 TargetEnvironment=Unity
diff --git a/sysui/desktop/menus/writer.desktop b/sysui/desktop/menus/writer.desktop
index 6d2274e..888b382 100755
--- a/sysui/desktop/menus/writer.desktop
+++ b/sysui/desktop/menus/writer.desktop
@@ -10,8 +10,8 @@ Name=%PRODUCTNAME Writer
 GenericName=Word Processor
 Comment=Create and edit text and graphics in letters, reports, documents and Web pages by using Writer.
 InitialPreference=5
-X-Ayatana-Desktop-Shortcuts=New
-[New Shortcut Group]
+X-Ayatana-Desktop-Shortcuts=X-New
+[X-New Shortcut Group]
 UnityQuicklist=New Document
 Exec=libreoffice --writer %U
 TargetEnvironment=Unity
-- 
1.7.10.2

